src/lib/Resources/config/container/solr/aggregation_result_extractors.yml

```
@@ -146,22 +146,22 @@ services:
       - { name: ibexa.search.solr.query.content.aggregation.result.extractor }
       - { name: ibexa.search.solr.query.location.aggregation.result.extractor }
 
-  ezplatform.search.solr.query.common.aggregation_result_extractor.location_children_term:
+  ibexa.solr.query.common.aggregation_result_extractor.location_children_term:
     class: Ibexa\Solr\ResultExtractor\AggregationResultExtractor\TermAggregationResultExtractor
     arguments:
       $aggregationClass: 'Ibexa\Contracts\Core\Repository\Values\Content\Query\Aggregation\Location\LocationChildrenTermAggregation'
       $keyMapper: '@Ibexa\Solr\ResultExtractor\AggregationResultExtractor\TermAggregationKeyMapper\LocationAggregationKeyMapper'
     tags:
       - { name: ibexa.search.solr.query.location.aggregation.result.extractor }
 
-  ezplatform.search.solr.query.common.aggregation_result_extractor.subtree_term.nested:
+  Ibexa\Solr\ResultExtractor\AggregationResultExtractor\NestedAggregationResultExtractor:
```

```suggestion
  ibexa.solr.query.common.aggregation_result_extractor.subtree_term.nested:
```
Ibexa\Solr\ResultExtractor\AggregationResultExtractor\NestedAggregationResultExtractor could be used in other context by design